src/utils/common.js

```diff
@@ -3,4 +3,19 @@ const calculateColumnSpan = (imageCount, imagesPerRow) => {
   return totalColumns
 }
 
-export { calculateColumnSpan }
+function siteIDToName(siteID) {
+  const siteIDMap = {
```

Comment:
We may want to get this mapping from the backend and set it in the pinia store in the frontend. The backend could query ConfigDB and keep a basic mapping of sites to locations in the redis cache. That also makes it extensible in the future in case we want to add any additional information.

Reply:
can't we have the Frontend query ConfigDB directly and store it in Pinia? That seems more direct and more easily extensible

Reply:
Unfortunately ConfigDB is hosted within the LCO internal VPN, so when a client outside of the VPN or LCO building wants to query configDB, it needs to happen via a service that runs inside the VPN.